conf/ansible/playbooks/es-configure.yml

```
@@ -0,0 +1,64 @@
+---
+- name: Configure Elasticsearch
+  hosts: "{{ target | default('localhost') }}"
+  become: false
+
+  pre_tasks:
+    - name: Source all env variables
+      ansible.builtin.include_tasks:
+        file: "{{ playbook_dir }}/../tasks/load-envs.yml"
+
+  tasks:
+    - name: Check if environment is staging or dev
+      ansible.builtin.set_fact:
+        is_staging_or_local: "{{ env == 'staging' or env == 'local' }}"
+      tags: always
+
+    - name: Ensure Elasticsearch is accessible (production)
+      ansible.builtin.uri:
+        url: "{{ es_api_scheme }}://{{ es_api_host }}:{{ es_api_port }}/_cluster/health"
+        method: GET
+        status_code: 200
+        headers:
+          Content-Type: "application/json"
+      register: es_info
+      until: es_info.status == 200
+      retries: 6
+      delay: 10
+      when: not is_staging_or_local | bool
+      tags: always
+
+    - name: Verify cluster health for all nodes (staging/local)
```

```suggestion
    - name: Ensure Elasticsearch is accessible (staging or local)
```

Since the task is doing the same thing, why not use similar name? Or why are we verifying "cluster" in PROD but individual nodes in staging/local?